bin/run_miniQMCPack.sh

```diff
@@ -0,0 +1,72 @@
+#! /usr/bin/env bash
+
+# run_miniQMCPack.sh - runs several MiniQMC binaries
+# The user should set at least these two variables:
+# MQMC_BUILD_PREFIX       miniqmc build prefix (default: ~/miniqmc_build)
+# MQMC_SOURCE_DIR         path to miniqmc sources (default: .)
+#
+# The user can set different variables to control execution.
+# MQMC_OMP_NUM_THREADS    how many OpenMP threads should be used (default 64)
+# MQMC_NUM_BUILD_PROCS    how many processes to build miniqmc (default 32)
+# ROCM_INSTALL_PATH       top-level ROCm install directory (default: /opt/rocm-5.3.0)
+
+# --- Start standard header to set AOMP environment variables ----
+realpath=`realpath $0`
+thisdir=`dirname $realpath`
+. $thisdir/aomp_common_vars
+# --- end standard header ----
+
+# Default ROCm installation
+: ${ROCM_INSTALL_PATH:=/opt/rocm-5.3.0}
```

Comment:
Rename to just ROCM (since we aren't installing ROCm in this script)...and default to /opt/rocm (no version number...which is a symlink to the installed version)